### PR TITLE
tesseract-ocr(4.1.1): fix absolute paths in .pc file

### PR DESCRIPTION
**Pull request by jannick0**
Replace the line of the .pc line (for MINGW64)
```
  Libs: -L${libdir} -ltesseract -LD:/msys64/mingw64/lib -larchive -LD:/msys64/mingw64/lib -lcurl
``` 
by
```
  Libs: -L${libdir} -ltesseract -L/mingw64/lib -larchive -L/mingw64/lib -lcurl
```

### CODE REVIEW

File: mingw-w64-tesseract-ocr/PKGBUILD [hunk 1]
```diff
@@ -19,15 +19,12 @@ depends=(${MINGW_PACKAGE_PREFIX}-cairo
...
-        001-proper-include-thread.patch)
+        https://github.com/tesseract-ocr/tessdata_fast/raw/master/osd.traineddata)
 sha256sums=('2a66ff0d8595bff8f04032165e6c936389b1e5727c3ce5a27b3e059d218db1cb'
-            '9cf5d576fcc47564f11265841e5ca839001e7e6f38ff7f7aacf46d15a96b00ff'
-            '0c8eec0552e9d3af93e84f594e71cfce0e996c630bddc6a0c2ca509f516696c9')
+            '9cf5d576fcc47564f11265841e5ca839001e7e6f38ff7f7aacf46d15a96b00ff')
 
 prepare() {
   cd "${srcdir}/tesseract-${pkgver}"
-  patch -p1 -i ${srcdir}/001-proper-include-thread.patch
```
Comment (lazka) [inline mingw-w64-tesseract-ocr/PKGBUILD:30]: Why did you remove this?

Comment (jannick0) [reply mingw-w64-tesseract-ocr/PKGBUILD:30]: Straight from the commit message:
```
* mingw-w64-tesseract-ocr/PKGBUILD:
  [...]
  - remove references to patch 001-proper-include-thread.patch having no
    effect.
```
The patch for a previous tesseract version became obsolete since the issue was patched in the source repository.  Could be easily checked there.

Comment (lazka) [reply mingw-w64-tesseract-ocr/PKGBUILD:30]: ok, thanks, I assumed the PR message was the same as the commit message.

Comment (jannick0) [reply mingw-w64-tesseract-ocr/PKGBUILD:30]: Yes - this was my first commit message which I should have changed after figuring out how to solve the issues of the very first commit.

Many thanks!
